packages/ui/src/molecules/AggregateRating/AggregateRating.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { forwardRef } from 'react'
+import type { FC, PropsWithChildren } from 'react'
+
+import List from '../../atoms/List'
+import type { ListProps } from '../../atoms/List'
+
+export interface AggregateRatingProps extends ListProps<'ul'> {
```

Comment:
Since you allow the variant props, the list can change from ul to another HTML element.
What do you think about letting it be generic?

```suggestion
export interface AggregateRatingProps<Variant extends ListVariant = 'ul'> extends ListProps<Variant> {
```